gravitee-apim-gateway/gravitee-apim-gateway-services/gravitee-apim-gateway-services-sync/src/main/java/io/gravitee/gateway/services/sync/process/repository/synchronizer/organization/FlowAppender.java

```diff
@@ -40,13 +40,13 @@ public class FlowAppender {
     public OrganizationDeployable appends(final OrganizationDeployable organizationDeployable) {
         List<String> shardingTags = gatewayConfiguration.shardingTags().orElse(null);
         if (shardingTags != null && !shardingTags.isEmpty()) {
-            filterFlows(organizationDeployable.organization());
+            filterFlows(organizationDeployable.reactableOrganization());
         }
         return organizationDeployable;
     }
 
-    private void filterFlows(final Organization organization) {
-        List<Flow> filteredFlows = organization
+    private void filterFlows(final ReactableOrganization reactableOrganization) {
```

Comment:
I'd rather this method return the list and the caller set it

Reply:
I just renamed Organization to ReactableOrganization to avoid confusion with the class from the model. I don't want to do more than require for the feature. (Also all appenders are working this way)